easier_net/sier_net.py

```diff
@@ -12,24 +12,26 @@
 
 from network import SierNet
 
-
-class SierNetEstimator:
+class SierNetEstimator: 
     """
     This first a single sparse-input hierarchical network (SIER-net).
     """
     def __init__(
         self,
-        n_inputs: int,
-        input_filter_layer: bool,
-        n_layers: int,
-        n_hidden: int,
-        n_out: int,
-        full_tree_pen: float,
-        input_pen: float,
-        batch_size: int,
-        num_classes: int,
-        weight: list,
-        connection_pen: float = 0,
+        n_inputs=4,
+        input_filter_layer=False,
```

Comment:
because you've added a lot of defaults, i think there are weird parts in the code that are failing silently. for example, `batch_size` is getting set to 3 even though it probably shouldnt be.